mbpfan.spec

```diff
@@ -0,0 +1,68 @@
+Name:           mbpfan
+URL:            https://github.com/dgraziotin/mbpfan
+License:        GPLv3
+Group:          System Environment/Daemons
+Version:        %{SOURCE_VERSION}
+Release:        1
+Summary:        A simple daemon to control fan speed on all MacBook/MacBook Pros (probably all Apple computers) for Linux 3.x.x and 4.x.x
+Source:         v%{version}.tar.gz
+
+%description
+This is an enhanced version of Allan McRae mbpfan
+
+mbpfan is a daemon that uses input from coretemp module and sets the fan speed using the applesmc module. This enhanced version assumes any number of processors and fans (max. 10).
+
+It only uses the temperatures from the processors as input.
+It requires coretemp and applesmc kernel modules to be loaded.
+It requires root use
+It daemonizes or stays in foreground
+Verbose mode for both syslog and stdout
+Users can configure it using the file /etc/mbpfan.conf
+
+%prep
+%setup -q -n %{name}-%{version}
+
+%build
+make
+
+%install
+install -D -m755 bin/mbpfan $RPM_BUILD_ROOT/usr/sbin/mbpfan
+install -D -m644 mbpfan.conf $RPM_BUILD_ROOT/etc/mbpfan.conf
+install -D -m644 mbpfan.service $RPM_BUILD_ROOT/usr/lib/systemd/system/mbpfan.service
+
+%clean
+rm -rf $RPM_BUILD_ROOT
+
+%post
+%systemd_post mbpfan.service
+# If it is a first installation then autoconfigure daemon
+if [[ ${1} == 1 ]]; then
+  min_fan_speed="$(cat /sys/devices/platform/applesmc.768/fan*_min| uniq| sort |head -1)"
+  max_fan_speed="$(cat /sys/devices/platform/applesmc.768/fan*_max| uniq| sort |tail -1)"
```

Comment:
Can we eliminate these if we merge #137?

Reply:
These lines will detect automatically the minimum and maximum fan speeds like the tutorial
<https://ineed.coffee/3838/a-beginners-tutorial-for-mbpfan-under-ubuntu/> suggests.
The preconfigured settings are not ideal for all Macs.

Reply:
Right, I suggest we detect the fan speeds in `mbpfan` itself instead of the systemd script.

Reply:
It could be a good idea. Please note that this two lines not belong to a systemd script but to a RPM scriptlet that is executed only on the first install of the package (not on update).

Reply:
I added min and max fan speed detection in 967d9789bb00ade24a71ca7f852aecee3b96419c to move this along.  Could you rebase this pull request and remove the spec detection?  Let's defer max temperature for now.

Reply:
@codutti Any progress on this pull request?  Should I release 2.1.0 without it?

Reply:
Sorry for the delay. I'm busy with my work. Could you give me a couple of days?

Reply:
Here I am. I've re-synced my fork with origin. Removed auto configured values on post-install script. I also commented the auto-detected values (min and max fan speeds) from config file otherwise the auto detection values will be always overwritten by default config files.
The commit is in my fork in https://github.com/codutti/mbpfan/commit/6523132531f55f1d42075d206608d5ba5b6d9245
It is fine?